video/out/hwdec/dmabuf_interop_gl.c

```diff
@@ -55,6 +74,11 @@ typedef void *EGLImageKHR;
 #define EGL_DMA_BUF_PLANE3_MODIFIER_LO_EXT 0x3449
 #define EGL_DMA_BUF_PLANE3_MODIFIER_HI_EXT 0x344A
 
+const uint32_t decomposed_nv12_fourcc[] = {
```

Comment:
`const static`
but also this is not referenced anywhere

Reply:
Yeah. This was me starting to do a lookup table approach and then I switched to the switch/case.